compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -294,13 +308,16 @@ precBinding = 0
 precParam = 1
 
 pPrintAndKind :: Pretty a => PrettyLevel -> Rational -> (a, Kind) -> Doc ann
-pPrintAndKind lvl prec (v, k) = case k of
-    KStar -> pPrintPrec lvl 0 v
-    _ -> maybeParens (prec > precBinding) (pPrintPrec lvl 0 v <-> docHasType <-> kind_ (pPrintPrec lvl 0 k))
+pPrintAndKind lvl prec (v, k)
+  | levelHasKinds lvl = maybeParens (prec > precBinding) $
```

Comment:
Your PR description says that you no longer emit kind annotations if the kind is start but this change seems to be doing exactly the opposite? We special-cased star before and did not emit the kind annotation whereas now we no longer special-case it.

Reply:
Looks like there are too many negations in my PR description. It says
> * Kind annotations are no longer omitted when the kind is `*` (star).

Meaning, we now emit kind annotations even when the kind is `*`.

Reply:
I misread omitted as emitted :facepalm: